drupal/sites/all/features/boinc_standard/boinc_standard.strongarm.inc

```diff
@@ -2049,21 +2049,9 @@ After logging in, you will be redirected to !edit_uri so you can change your pas
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
   $strongarm->name = 'user_mail_register_no_approval_required_body';
-  $strongarm->value = '!username,
-
-Thank you for registering at !site. You may now log in to !login_uri using the following username and password:
-
-username: !username
-password: !password
-
-You may also log in by clicking on this link or copying and pasting it in your browser:
-
-!login_url
-
-This is a one-time login, so it can be used only once.
-
-After logging in, you will be redirected to !edit_uri so you can change your password.
+  $strongarm->value = '!mailto,
 
+Thank you for registering at !site. You may now log in to !login_uri using the following email address and your passphrase.
```

Comment:
This section of code seems to be missing the "following email address", as all that follows is the signature line of the message.

Reply:
Thanks for catching this. I'll change it and update the PR.